### PR TITLE
Add 11.8 release notes

### PR DESCRIPTION
**Pull request by lzdanski**

### CODE REVIEW

File: astro/runtime-release-notes.md [hunk 1]
```diff
@@ -29,6 +29,17 @@ If you're upgrading to receive a specific change, ensure the release note for th
 
 :::
 
+## Astro Runtime 11.8.0
+
+- Release date: August 8, 2024
```
Comment (utkarsharma2) [inline astro/runtime-release-notes.md:34]:
```suggestion
- Release date: August 9, 2024
```

File: astro/runtime-release-notes.md [hunk 2]
```diff
@@ -29,6 +29,17 @@ If you're upgrading to receive a specific change, ensure the release note for th
...
+- Airflow version: 2.9.3
+
+### Additional improvements
+
+- Fixed the Tasks API endpoint for scenarios when a DAG doesn't have a `start_date` ([#40878](https://github.com/apache/airflow/pull/40878)
+- Added validation for the project URL that comes from installed providers, before displaying the URL in views ([#40933](https://github.com/apache/airflow/pull/40933))
+- Upgraded the minor and patch versions of some Astro open source provider packages. See [Astro Runtime 11.8.0 provider packages](https://www.astronomer.io/docs/astro/runtime-provider-reference#astro-runtime-1180)
```
Comment (utkarsharma2) [inline astro/runtime-release-notes.md:41]:
```suggestion
- Upgraded the minor and patch versions of some open-source provider packages. See [Astro Runtime 11.8.0 provider packages](https://www.astronomer.io/docs/astro/runtime-provider-reference#astro-runtime-1180)
```

**Review by utkarsharma2 — APPROVED** [review]
Few minor changes, LGTM.

File: astro/runtime-release-notes.md [hunk 3]
```diff
@@ -29,6 +29,17 @@ If you're upgrading to receive a specific change, ensure the release note for th
...
 :::
 
+## Astro Runtime 11.8.0
+
+- Release date: August 9, 2024
+- Airflow version: 2.9.3
+
+### Additional improvements
+
+- Fixed the Tasks API endpoint for scenarios when a DAG doesn't have a `start_date` ([#40878](https://github.com/apache/airflow/pull/40878)
```
Comment (lzdanski) [inline astro/runtime-release-notes.md:39]:
```suggestion
- Fixed the Tasks API endpoint for scenarios when a DAG doesn't have a `start_date` ([#40878](https://github.com/apache/airflow/pull/40878))
```

**Pull request merged**